client/js/components/month/month.js

```diff
@@ -1,41 +1,43 @@
 /** @jsx React.DOM */
 var React = require('react');
-var MonthBoard = require('./month-board.js');
+var MonthBoard = require('./month-board');
 var MonthStore = require('../../stores/month-store');
 var MonthHeader = require('./month-header');
 var ViewActionCreator = require('../../actions/view-action-creator');
+var Day = require('../day/day');
+var MonthDayPreview = require('./month-day-preview');
+var moment = require('moment');
 
 var Month = React.createClass({
   getInitialState: function () {
     return {
-      'week1': [{date: '', numEvents: 0, status: ''}],
-      'week2': [{date: '', numEvents: 0, status: ''}],
-      'week3': [{date: '', numEvents: 0, status: ''}],
-      'week4': [{date: '', numEvents: 0, status: ''}]
+      startDate : '2015-05-10',
```

Comment:
What's the meaning of this start date? Should be able to just use the current day.